src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -94,54 +103,49 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     setCurrentIndex(index);
   };
 
-  const mainClass = React.useCallback(() => {
-    switch (type) {
-      case 'borderless':
-        return 'iui-tabs-borderless';
-      case 'pill':
-        return 'iui-tabs-pill';
-      default:
-        return 'iui-tabs-horizontal';
-    }
-  }, [type]);
-
   return (
-    <>
+    <div className='iui-tabs-wrapper'>
       <ul
         className={cx(
-          mainClass(),
-          { 'iui-green': color === 'green' },
+          'iui-tabs',
+          {
+            [`iui-${type}`]: !!type,
+            'iui-green': color === 'green',
```

Comment:
Borderless tabs seem to be influenced the most. There is a big size change in that, so everyone using those should see the diff.. It was decided that previously we had too big ones?

Reply:
We had a lot of questionable things in the previous version, so Jon decided new Tabs should be more consistent.
![image](https://user-images.githubusercontent.com/9084735/123802460-531eaa00-d8b9-11eb-9cd7-92acd54ac453.png)

I think once we provide `size=large`, people could have an equivalent to the old height.